content/docs/tutorials/_index.md

```diff
@@ -10,7 +10,14 @@ menu:
 aliases: ["/docs/reference/tutorials/"]
 ---
 
-Please select from the following cloud providers or tutorials:
+From VMs to Serverless to Containers, our collection of tutorials below represent a great jumping off point for cloud programmers of all shapes and sizes. You can deploy each of these tutorials as they are, or you can use them as a starting off point to deploy your own applications and infrastructure.
+
+> **First time using Pulumi?** Visit our [Getting Started Guide]({{< relref "/docs/get-started" >}}) for a quick walk through of creating your first Pulumi program and the basic CRUD operations of the Pulumi CLI.
```

Comment:
Suggest removing both of these blockquotes, as they'd interfere with the validity of the test results, depending on how "success" is measured. (That is, if you saw this page first, then clicked Getting Started and ended up going through the flow, you should be excluded from the test results.)